File: app/src/main/java/org/schulcloud/mobile/data/model/News.java
package org.schulcloud.mobile.data.model;

import io.realm.RealmModel;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.RealmClass;

@RealmClass
public class News implements RealmModel {
    @PrimaryKey
    public String _id;
    public String schoolId;
    public String title;
    public String content;
    public String createdAt;
}

File: app/src/main/java/org/schulcloud/mobile/models/news/News.kt
package org.schulcloud.mobile.models.news

import com.google.gson.annotations.SerializedName
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import org.schulcloud.mobile.models.base.HasId

open class News : RealmObject(), HasId {

    @PrimaryKey
    @SerializedName("_id")
    override var id: String = ""

    var schoolId: String? = null
    var title: String? = null
    var content: String? = null
    var createdAt: String? = null
}
